tests/core/test_api.py

```diff
@@ -214,3 +215,47 @@ def mock_get_db_health(url: str) -> str:
     monkeypatch.setattr(main, "get_db_health", mock_get_db_health)
     response = test_client.get(test_config.cli.server_url + API_PREFIX + "/health")
     assert response.status_code == expected_status_code
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "sample_string", ["test1:'as?/<>.,", "!@#$%^&*()-_+=", "1a2b3c4d5e6F7G8H9I0J"]
+)
+def test_obscure_string(sample_string: str) -> None:
+    obscured_string = obscure_string(sample_string)
+    assert sample_string == unobscure_string(obscured_string)
```

Comment:
What do you think of this being in its own file? routes/test_util.py. feels like it shouldn't be tested here.

Reply:
Ya it feels unnatural here. I saw some other items that sort of overlapped but it feels like we should split this out 👍🏽

Reply:
Moved as suggested